girder_worker/format/integer/validate_json.json

```diff
@@ -0,0 +1,7 @@
+{
+    "inputs": [{"name": "input", "type": "integer", "format": "json"}],
+    "outputs": [{"name": "output", "type": "boolean", "format": "boolean"}],
+    "script": "output = isinstance(input, (str, unicode))",
+    "extensions": ["integer-json"],
```

Comment:
`extensions` are just a way for Flow to infer what type/format something is. If there are conflicts (i.e. tables and graphs could both theoretically be stored as a csv) then Flow forces the user to resolve the conflict.